DataProcessor/DataUnpacker.cpp

```diff
@@ -40,7 +40,7 @@ DataUnpacker::DataUnpacker(QObject *parent) : QObject(parent)
 {
     FILE* fp = fopen("../sc1-driver-io/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
     if(fp == 0) {
-        fp = fopen("../solar-car-dashboard/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
+        fp = fopen("../sc1-driver-io/sc1-data-format/format.json", "r"); // NOTE: Windows: "rb"; non-Windows: "r"
```

Comment:
Should be solar-car-dashboard instead of sc1-driver-io here. This line runs if sc1-driver-io wasn't found in the previous attempt to open the json file